recipe/gen_patch_json.py

```diff
@@ -281,6 +283,25 @@
     'openmpi-4.0.1-hc99cbb1_2': '10.12',
 }
 
+
+def _add_broken_to_removals(subdir):
+    global REMOVALS
+    r = requests.get(
+        "https://conda-static.anaconda.org/conda-forge/"
```

Comment:
Shouldn't this be `conda-web` ?

Reply:
Idk the difference between these things. I asked in the core gitter.

Reply:
We should make sure anaconda is ok with us using conda-web here too